### PR TITLE
Upgrade rails

### PR DESCRIPTION
**Pull request by HParker**
I have a couple more failing tests, but I want to see what Circle says. kinda sensing they are environment things.

### CODE REVIEW

**Comment by HParker** [comment]
@XanderStrike BAM! new rails!

File: app/controllers/encodes_controller.rb [hunk 1]
```diff
@@ -10,8 +10,8 @@ def retag
 
     old_movie = Movie.find_by_id(encode.movie_id)
 
-    `wget https://d3gtl9l2a4fn1j.cloudfront.net/t/p/w500/#{ info.poster_path } -O ./public/posters/#{info.id}.jpg -b -q`
-    `wget http://image.tmdb.org/t/p/w1000/#{ info.backdrop_path } -O ./public/backdrops/#{info.id}.jpg -b -q`
+    # `wget https://d3gtl9l2a4fn1j.cloudfront.net/t/p/w500/#{ info.poster_path } -O ./public/posters/#{info.id}.jpg -b -q`
```
Comment (XanderStrike) [inline app/controllers/encodes_controller.rb]: Accident?

Comment (HParker) [inline app/controllers/encodes_controller.rb]: oh, maybe? the fact that I don't know probably means that it was.

Comment (HParker) [inline app/controllers/encodes_controller.rb]: tests still pass with these lines back in, so I put them back.

**Pull request merged**